github/src/webhook.rs

```diff
@@ -31,21 +31,30 @@ impl Webhook {
         // Skip validation if no key
         // FIXME: Refactor code so it's required
         if key.is_none() {
-            warn!("[webhook {}] No secret specified; signature ignored", self.delivery_id);
-            return true
+            warn!(
+                "[webhook {}] No secret specified; signature ignored",
+                self.delivery_id
+            );
+            return true;
         }
 
         if let Some(ref signature) = self.signature_256 {
-            let hash = hex::encode(hmacsha1::hmac_sha1(key.unwrap(), &self.body));
+            let hash = hex::encode(hmac_sha256::HMAC::mac(&self.body, key.unwrap()));
```

Comment:
Can this commit be squashed into the previous one? so that you don't introduce the sha256 sig and then immediately fix it in the next commit?

Reply:
Yeah I can fix that

Reply:
Yeah I can fix that